src/y0/algorithm/falsification.py

```diff
@@ -90,24 +90,16 @@ def __init__(self, failures, evidence):
         def __getitem__(self, i): return self._failures[i]
         def __len__(self): return len(self._failures)
         def __repr__(self): return repr(self._failures) + "+evidence"
-    
-    wrapper = lambda v, desc: v
-    if verbose: 
-        try: 
-            import tqdm
-            wrapper = tqdm.tqdm
-        except:
-            print("TQDM not installed, verbose mode not supported")
-    
+
     all_nodes = set(G.vertices)
     all_pairs = combinations(all_nodes, 2)
 
-    to_test = [(a, b, given) for a,b in wrapper(all_pairs, desc="Checking d-separation")
+    to_test = [(a, b, given) for a,b in tqdm(all_pairs, desc="Checking d-separation")
```

Comment:
I would like to leave the verbose flag in place, only using tqdm when verbose is requested.  
Moving tqdm to the the top is a good idea, since it is a project dependency elsewhere.

Reply:
tqdm has a keyword argument in it you can use instead to disable it called `disable`

Reply:
Was not aware of that.  Great!